ALUM/ALUM/Views/CalendlyBooking/CalendlyBooking.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  CalendlyBooking.swift
+//  ALUM
+// This is a file used to store a dummy button for testing functionality of
+// booking an event via calendly. Note that the majority of the code in here will be transferred to
+// the profile view code once the PR has been updated
+//
+//  Created by Adhithya Ananthan on 4/12/23.
+//
+
+import Foundation
+import SwiftUI
+import WebKit
+
+
+struct CalendlyBooking: View  {
+    @State public var showWebView = false
+    // This should be updated to use the mentors link
+    public var urlString: String = "http://localhost:3000/calendly?url=https://calendly.com/aananthanregina/test"
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            Button {
+                showWebView.toggle()
+                print("Button Pushed")
+            }
+            label: {
+                Text("View My Calendly")
+                .font(Font.custom("Metropolis-Regular", size: 17, relativeTo: .headline))
+            }
+            .sheet(isPresented: $showWebView) {
+                CalendlyView(url: URL(string: urlString)!, menteeId: "6431b99ebcf4420fe9825fe3", mentorId: "6431b9a2bcf4420fe9825fe5")
```

Comment:
We should remove the need for passing IDs to POST session cauz we are sending auth token anyways. In backend, use auth token to fill in the menteeID and then use the pairing schema to get the mentorID

https://github.com/TritonSE/ALUM-Mobile-Application/issues/93